salon_creator/lib/screens/salon_create_application_screen.dart

```diff
@@ -0,0 +1,315 @@
+import 'package:flutter/material.dart';
+import 'package:salon_creator/common/email.dart';
+import 'package:salon_creator/widgets/customButton.dart';
+import 'package:salon_creator/widgets/customDialog.dart';
+import 'package:salon_creator/widgets/customLabel.dart';
+
+class ApplicationScreen extends StatefulWidget {
+  const ApplicationScreen({Key key}) : super(key: key);
+
+  @override
+  _ApplicationScreenState createState() => _ApplicationScreenState();
+}
+
+class _ApplicationScreenState extends State<ApplicationScreen> {
+  final contentController = TextEditingController();
+  final firstNameController = TextEditingController();
+  final lastNameController = TextEditingController();
+  final phoneNumberController = TextEditingController();
+
+  bool emptyChecker() {
```

Comment:
```suggestion
  bool _noFieldsEmpty() {
```
Making the function private and more intuitive in name